platform/x11/os_x11.cpp

```diff
@@ -1756,6 +1756,18 @@ void OS_X11::request_attention() {
 	XFlush(x11_display);
 }
 
+void *OS_X11::get_native_handle(int p_handle_type, int p_index) {
+	/* Godot 3.2 does not have multiple window support so we ignore p_index */
+	switch (p_handle_type) {
+		case APPLICATION_HANDLE: return NULL; // Do we have a value to return here?
+		case DISPLAY_HANDLE: return &x11_display;
+		case WINDOW_HANDLE: return &x11_window;
```

Comment:
Any reason to use addresses? `x11_display` is already a pointer type (X11::Display *) and `x11_window` is `XID` == `unsigned long`, both should fit by value.

Reply:
I'll change it, I wasn't sure because I don't do much with Linux so I did that one blind. We are returning a signed int because I think that is what GDNative / Variant type support but hopefully that will not pose a problem.